docs/learn/documentation/versioned/operations/monitoring.md

```diff
@@ -369,6 +369,7 @@ All \<system\>, \<stream\>, \<partition\>, \<store-name\>, \<topic\>, are popula
 | | expired-preferred-host-requests | Number of expired resource-requests-for -preferred-host received by the cluster manager. |
 | | expired-any-host-requests | Number of expired resource-requests-for -any-host received by the cluster manager. |
 | | host-affinity-match-pct | Percentage of non-expired preferred host requests. This measures the % of resource-requests for which host-affinity provided the preferred host. |
+| | \<containerId\>-failure-count | Number of times a container identified by containerId has failed |
```

Comment:
I believe we decided to use "processorId" for 0,1,2..

Reply:
that lingo is used internally in code as the naming conventions for javadocs, this is public-facing metrics page where we do not need to have context between processorId and containerId